brainscore/model_interface.py

```diff
@@ -117,6 +117,32 @@ class Task:
                   - choice        (choice) object 'dog' 'cat' 'chair' 'flower' 'plane'
         """
 
+        odd_one_out = 'odd_one_out'
+        """
+        Predict the odd-one-out elements for a list of triplets of stimuli. 
+
+        The model must be supplied with a list of triplets of stimuli. Then it goes through the following steps:
+        1. Computation of representations for each unique stimulus within the triplet list.
+        2. Calculation of similarity scores between the activation pairs for every triplet.
+        3. Identification of the stimulus with the lowest similarity, designating it as the odd-one-out
+        4. Returning the index (0, 1, or 2) corresponding to the odd-one-out element in each triplet."
```

Comment:
```suggestion
        The model must be supplied with a list of stimuli where 
        all three consecutive stimuli are considered to form a triplet.
        The model is expected to output a one-dimensional assembly with each value corresponding 
        to the index (`0`, `1`, or `2`) of the triplet element that is different from the other two.
```
I like the steps documentation, but this should go into the implementation in model-tools. This is just one possible documentation, so the interface should not declare what implementation the model has to use.

Reply:
"all three consecutive stimuli are considered to form a triplet" is either not what we want or easy to misunderstand. I would like to keep the list of triplets instead of your flat list.

Reply:
I'm worried the list of triplets will cause clashes with other functionality. Imagine for instance we want to record from a model while it performs the odd-one-out task. In that case we would run:
```
model.start_task(BrainModel.Task.odd_one_out)
model.start_recording(BrainModel.RecordingTarget.IT, ...)
model.look_at(stimuli)
```
If `stimuli` is a list of triplets, the code to record from the model will not be able to deal with it since it expects a flat list.